### PR TITLE
Added Network Watcher PS cmdlets and tests

### PR DESCRIPTION
**Pull request by irrogozh**
## Description (NOT FOR MERGE)

Tests For expressRoute are failing in playback mode. Discussed this issue with Ding Hun, who made changes for expressRoute in SDK.
---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md).**
- [x] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md).**
- [x] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.
- [x] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [x] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://gist.github.com/markcowl/338e16fe5c8bbf195aff9f8af0db585d#what-is-the-change).
- [x] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [x] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [x] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [x] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@irrogozh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (irrogozh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by cormacpayne** [comment]
@azuresdkci add to whitelist

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 1]
```diff
@@ -48,6 +48,28 @@
     <DelaySign>true</DelaySign>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: Please do not add x64 targets

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 2]
```diff
@@ -73,8 +95,16 @@
...
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.8.1.0-preview\lib\net45\Microsoft.Azure.Management.Network.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.Compute">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Compute.13.1.0-prerelease\lib\net45\Microsoft.Azure.Management.Compute.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=8.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.8.2.0-preview\lib\net45\Microsoft.Azure.Management.Network.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.Azure.Management.Storage, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: Please do not add storage and network management depednencies, why do you need these?

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherFlowLogStatusCommand.cs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherFlowLogStatusCommand.cs]: @irrogozh (same comment for `NetworkWatcherName`) there should only be one parameter with the `ValueFromPipeline` attribute in a parameter set, and it should take in an object, not just the property of an object. Consider using `ValueFromPipelineByPropertyName` if it applies in this scenario

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherNextHopCommand.cs [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherNextHopCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherSecurityGroupViewCommand.cs [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherSecurityGroupViewCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherTopologyCommand.cs [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherTopologyCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherTroubleshootingResult.cs [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherTroubleshootingResult.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/GetAzureNetworkWatcherPacketCaptureCommand.cs [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/GetAzureNetworkWatcherPacketCaptureCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/NewAzureNetworkWatcherPacketCaptureCommand.cs [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/NewAzureNetworkWatcherPacketCaptureCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/RemoveAzureNetworkWatcherPacketCaptureCommand.cs [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/RemoveAzureNetworkWatcherPacketCaptureCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/StopAzureNetworkWatcherPacketCaptureCommand.cs [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The packet capture name.")]
+        [ValidateNotNullOrEmpty]
+        public virtual string PacketCaptureName { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+            if (ParameterSetName.Contains("SetByResource"))
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/StopAzureNetworkWatcherPacketCaptureCommand.cs]: @irrogozh this cmdlet implements `SupportsShouldProcess`, but there is no `ConfirmAction` method wrapping these calls

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/StopAzureNetworkWatcherPacketCaptureCommand.cs [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/StopAzureNetworkWatcherPacketCaptureCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/SetAzureNetworkWatcherConfigFlowLogCommand.cs [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/SetAzureNetworkWatcherConfigFlowLogCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/StartAzureNetworkWatcherResourceTroubleshootingCommand.cs [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/StartAzureNetworkWatcherResourceTroubleshootingCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherIPFlowCommand.cs [hunk 15]
```diff
@@ -0,0 +1,134 @@
...
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherIPFlowCommand.cs]: @irrogozh same comment about `ValueFromPipeline`

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/NewPacketCaptureFilterCommand.cs [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Remote port")]
+        public string RemotePort { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/NewPacketCaptureFilterCommand.cs]: @irrogozh this cmdlet implements `SupportsShouldProcess`, but the `ConfirmAction` method does not wrap the functionality

File: src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 17]
```diff
@@ -233,9 +247,21 @@
...
+    </None>
+    <None Include="ScenarioTests\NetworkWatcherAPITests.ps1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
     <None Include="ScenarioTests\PublicIpAddressTests.ps1">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkWatcherAPITests\TestGetSecurityGroupView.json" />
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkWatcherAPITests\TestGetTopology.json" />
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkWatcherAPITests\TestPacketCapture.json" />
```
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj]: Fixed

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj (hunk 2 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: Fixed

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj (hunk 1 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: Fixed

File: src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 18]
```diff
@@ -320,12 +346,14 @@
...
     </None>
-    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkInterfaceTests\TestNetworkInterfaceWithAcceleratedNetworking.json" >
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkInterfaceTests\TestNetworkInterfaceWithAcceleratedNetworking.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
     <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkInterfaceTests\TestNetworkInterfaceWithIpConfiguration.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkWatcherAPITests\TestVerifyIPFlow.json" />
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkWatcherTests\TestNetworkWatcherCRUD.json" />
```
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj]: Fixed

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 19]
```diff
@@ -73,8 +95,16 @@
...
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Storage.5.0.0-preview\lib\net45\Microsoft.Azure.Management.Storage.dll</HintPath>
```
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: Removed

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj (hunk 1 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: removed

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj (hunk 2 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: Removed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherSecurityGroupViewCommand.cs (hunk 5 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherSecurityGroupViewCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherFlowLogStatusCommand.cs (hunk 3 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherFlowLogStatusCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherNextHopCommand.cs (hunk 4 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherNextHopCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherTopologyCommand.cs (hunk 6 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherTopologyCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherTroubleshootingResult.cs (hunk 7 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherTroubleshootingResult.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/GetAzureNetworkWatcherPacketCaptureCommand.cs (hunk 8 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/GetAzureNetworkWatcherPacketCaptureCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/NewAzureNetworkWatcherPacketCaptureCommand.cs (hunk 9 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/NewAzureNetworkWatcherPacketCaptureCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/RemoveAzureNetworkWatcherPacketCaptureCommand.cs (hunk 10 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/RemoveAzureNetworkWatcherPacketCaptureCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/StopAzureNetworkWatcherPacketCaptureCommand.cs (hunk 12 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/StopAzureNetworkWatcherPacketCaptureCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/StopAzureNetworkWatcherPacketCaptureCommand.cs (hunk 11 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/StopAzureNetworkWatcherPacketCaptureCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/SetAzureNetworkWatcherConfigFlowLogCommand.cs (hunk 13 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/SetAzureNetworkWatcherConfigFlowLogCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/StartAzureNetworkWatcherResourceTroubleshootingCommand.cs (hunk 14 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/StartAzureNetworkWatcherResourceTroubleshootingCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherIPFlowCommand.cs (hunk 15 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherIPFlowCommand.cs]: Fixed

File: src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/NewPacketCaptureFilterCommand.cs (hunk 16 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/NewPacketCaptureFilterCommand.cs]: Removed SupportsShouldProcess

**Pull request closed without merging**